frontends/uhdm/UhdmAst.cc

```diff
@@ -555,7 +568,33 @@ void UhdmAst::process_module() {
 							  obj_h,
 							  [&](AST::AstNode* node) {
 								  if (node) {
+									if (node->type == AST::AST_INITIAL) {
+										auto module_child = current_node->children.begin();
+										for (;module_child != current_node->children.end(); module_child++) {
+											if ((*module_child)->type == AST::AST_ALWAYS) {
+												break;
+											}
+										}
+										if (module_child != current_node->children.end())
+										// Place AST_INITIAL before AST_ALWAYS
+											current_node->children.insert(module_child, 1, node);
+										else
+											add_or_replace_child(current_node, node);
+									} else if (node->type == AST::AST_ASSIGN) {
+										auto module_child = current_node->children.begin();
+										for (;module_child != current_node->children.end(); module_child++) {
+											if ((*module_child)->type == AST::AST_ALWAYS) {
+												break;
+											}
+										}
+										if (module_child != current_node->children.end())
+										// Place AST_ASSIGN before AST_ALWAYS
+											current_node->children.insert(module_child, 1, node);
+										else
+											add_or_replace_child(current_node, node);
+									} else {
```

Comment:
Why do we need this change? I think this shouldn't be needed.

Reply:
Right, this is not needed. It is now removed.